packages/components/src/Gesture/withListGesture.js

```diff
@@ -58,10 +58,12 @@ function onKeyDown(event, ref) {
 	switch (event.keyCode) {
 		case keycode.codes.down:
 			event.stopPropagation();
+			event.preventDefault();
```

Comment:
add `preventDefault()` to prevent time picker container from scrolling on every Up/Down pressing